app/dossierPrograms/dossierPrograms.view.html

```diff
@@ -89,12 +89,15 @@ <h3 class="section" id="{{ section.id }}">{{ section.displayName }}</h3>
                                 <td>{{ dataElement.displayDescription }}</td>
                                 <td>{{ dataElement.valueType }}</td>
                                 <td>
-                                    <select ng-if="dataElement.optionSet.options" class="form-control col-md-2">
-                                        <option selected>See options</option>
-                                        <option ng-repeat="option in dataElement.optionSet.options">
-                                            {{ option.displayName }}
-                                        </option>
-                                    </select>
+                                    <div ng-if="dataElement.optionSet.options">
+                                        <p>Option Set name: {{ dataElement.optionSet.name }}</p>
```

Comment:
Do we need `| translate` here? (and other literals in the file)

Reply:
Yes, an overlook by my part.